modules/apps/foundation/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/util/UADAnonymizerHelper.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.user.associated.data.util;
+
+import aQute.bnd.annotation.ProviderType;
+
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.User;
+
+/**
+ * @author Drew Brokke
+ */
+@ProviderType
+public interface UADAnonymizerHelper {
```

Comment:
I don't think we need an interface for the helper class. Why not just put the implementation here and make it available as a service directly?

Reply:
That should be fine, as long as we avoid making any fields a part of the API.